lib/hard_data.dart

```diff
@@ -55,13 +55,11 @@ const List<ReusableCard> icu = [
   ),
   ReusableCard(
     title: 'General Care',
-    description: '',
```

Comment:
do we want to keep the description property with the empty placeholder for future use?

Reply:
This is here because the vertical alignment of the text changes depending if description is null or not. When description is null, we want title to be centred. https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/blob/8c70decee808c35c293093ddc300720f1d663ebe/lib/widget/reusable_card.dart#L52